tests/test_api_query.py

```diff
@@ -8,7 +8,7 @@ def test_api_call():
     """
 
     api_interface = API_interfaces["carbonintensity.org.uk"]
-    response = cats.api_query.get_tuple(
+    response = cats.api_query.get_CI_forecast(
```

Comment:
That's calling `get_CI_forecast` (`get_tuple`) with its old signature

Reply:
Fixed in 76000f8fad3bd1979a97fb10b118b6daf5fe67a2